common/beerocks/bwl/dwpal/base_wlan_hal_dwpal.cpp

```diff
@@ -732,5 +925,11 @@ std::string base_wlan_hal_dwpal::get_radio_mac()
     return mac;
 }
 
+bool base_wlan_hal_dwpal::process_dwpal_nl_event(struct nl_msg *msg)
+{
+    LOG(DEBUG) << __func__ << "isn't implemented by derived and shouldn't be called";
```

Comment:
So why isn't it pure virtual? then it MUST be implemented by the derived class.

Reply:
It can't be pure virtual because it will only be implemented by the monitor class.

Which makes me wonder why this stuff is in the base class anyway, if it's only used for dwpal monitor.

Reply:
currently it's only used in the dwpal monitor
but since this can be used by all libraries with dwpal we didn't want to restrict it to just be available in the monitor